pkg/cclient/cclient.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"C"
+	"fmt"
+	"os"
+
+	"github.com/intel/afxdp-plugins-for-kubernetes/internal/uds"
+	"github.com/intel/afxdp-plugins-for-kubernetes/pkg/goclient"
+)
+
+func main() {
+	// Needed for cgo to generate the .h
+}
+
+var cleaner uds.CleanupFunc
+
+/*
+GetClientVersion is an exported version for c of goclient's GetClientVersion()
```

Comment:
```suggestion
GetClientVersion is an exported version for c of the goclient GetClientVersion()
```

same throughout the file... I don't think `goclient's` is grammatically correct